frontend/app/routes/$lang+/_public+/status+/index.tsx

```diff
@@ -125,19 +140,21 @@ export default function StatusChecker() {
       <Form method="post" noValidate>
         <input type="hidden" name="_csrf" value={csrfToken} />
         <div className="space-y-6">
-          <InputField id="sin" name="sin" label="Please enter your SIN" required />
-          <InputField id="code" name="code" label="Please enter your application code" required />
+          <InputField id="code" name="code" label={t('status:form.application-code-label')} helpMessagePrimary={t('status:form.application-code-description')} required />
+          <InputField id="sin" name="sin" label={t('status:form.sin-label')} required />
         </div>
         <Button className="mt-8" id="submit" variant="primary">
-          Check status
+          {t('status:form.submit')}
         </Button>
       </Form>
 
-      {actionData && (
-        <dd className="mt-8">
-          <dt className="font-semibold">Status:</dt>
-          <dd>{actionData.status ?? 'No application'}</dd>
-        </dd>
+      {actionData && actionData.clientFriendlyStatus && (
+        <ContextualAlert type={actionData.alertType ?? 'info'}>
+          <div>
+            <h2 className="mb-2 font-bold">Status</h2>
+            {getNameByLanguage(i18n.language, actionData.clientFriendlyStatus)}
+          </div>
+        </ContextualAlert>
       )}
```

Comment:
Use type narrowing to get the status if it's defined.

```suggestion
      {actionData && 'status' in actionData && (
        <ContextualAlert type={actionData.status.alertType}>
          <div>
            <h2 className="mb-2 font-bold">Status</h2>
            {getNameByLanguage(i18n.language, actionData.status)}
          </div>
        </ContextualAlert>
      )}
```